classes/core/PKPApplication.php

```diff
@@ -469,6 +469,11 @@ abstract public function getContextName(): string;
      */
     abstract public function getVersionDescriptorUrl(): string;
 
+    /**
+     * Get the help URL for this application.
+     */
+    abstract public function getHelpUrl(): string;
```

Comment:
This will probably be a fixed string

```suggestion
    abstract public static function getHelpUrl(): string;
```